src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1756,16 +1756,32 @@ private void visitTypeAlias(JSType registryType, TypedVar symbol) {
 
     private void visitTypeAlias(
         JSType registryType, String unqualifiedName, boolean emitNeverBrand) {
-      emit("type");
-      emit(unqualifiedName);
-      emit("=");
-      visitType(registryType, true, false);
+      visitTypeAliasHelper(registryType, unqualifiedName);
       // emit a brand to prevent accidental compatibility of values with an enum.
       if (emitNeverBrand) emit("&{clutzEnumBrand: never}");
       emit(";");
       emitBreak();
     }
 
+    private void visitTypeAlias(
+        JSType registryType, String unqualifiedName, Set<String> literalTypes) {
```

Comment:
add a comment that intentionally we always emit the brand here. Also, this is really only safe for string literal types, add that to the javadoc too.

Reply:
The more I think about it, I don't think this should be an overload of 'visitTypeAlias', but rather 'visitTypeAliasForMixedStringEnums'. I don't see how the function will be useful for other type alias declarations.

Reply:
Renamed the function and skipped the javadoc because the function name is more descriptive now.